src/NzbDrone.Core/MediaFiles/EpisodeImport/Augmenting/Augmenters/AugmentQuality.cs

```diff
@@ -0,0 +1,145 @@
+﻿using NLog;
+using NzbDrone.Core.MediaFiles.MediaInfo;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.Qualities;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.MediaFiles.EpisodeImport.Augmenting.Augmenters
+{
+    public class AugmentQuality : IAugmentLocalEpisode
```

Comment:
Augment: make (something) greater by adding to it; increase.
Aggregate: a whole formed by combining several (typically disparate) elements.

I see how both augment and aggregate apply in this case, I'm having a tough time choosing one over another. I think either works, but both sound more correct over 'Analyze'.

Reply:
In this case we have both... something that augments the Info by adding extracting data from the mediainfo etc. And something that aggregates the 4 EpisodeInfo instances into one 'truth'.

What about a compromise:
- `IAnalyzeLocalEpisode` generic interface
- `AugmentQualityFromMediaInfo`, does what it says, pulls quality info from MediaInfo.
- `AggregateQuality`, combines the info.

This would give us the opportunity to name instances to whatever specific purpose they have.
It also means we can split up the Augmentors into smaller dedicated pieces, and let the Aggregators combine it.

Reply:
I thought `Analyze` was a bit out of place, I went with `IAggregateLocalEpisode` and split the augmentation "gathers" for quality out to their own classes. There is some `Confidence` added to the augmentation results to allow overriding of lower confidence results.